README.md

```diff
@@ -85,6 +90,7 @@ None
   roles:
     - ansible-role-fluentd
   vars:
+    fluentd_flags: "{% if ansible_os_family == 'FreeBSD' %}-p {{ fluentd_plugin_dir }}{% elif ansible_os_family == 'Debian' %}{% elif ansible_os_family == 'RedHat' %}{% endif %}"
```

Comment:
i think it is better to add -p option when fluentd_plugins_to_install is not empty, or it should be documented that users need to set -p option when fluentd_plugins_to_install is not empty.